src/cpu/aarch64/brgemm/jit_brgemm_kernel.cpp

```diff
@@ -1876,7 +1876,16 @@ void jit_brgemm_kernel_t::bdb_loop() {
 }
 
 void jit_brgemm_kernel_t::generate() {
+    size_t simd_w_ = (brg.isa_impl == sve_512)
```

Comment:
Nit: sticking to `switch` statement is safer and will allow to catch an error when another length will be added.

Reply:
Thank you for the suggestion. The switch statement has been implemented as recommended.